Engine/Expressions/BinaryExpressionNode.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace OpenTap.Expressions
+{
+    /// <summary> Binary expressions AST node. This describes the application of a binary operator for example '*' in 5 * 4.</summary>
+    class BinaryExpressionNode : AstNode
+    {
+        /// <summary> Left side of the operation. </summary>
+        public AstNode Left;
```

Comment:
Again maybe more of a style thing. But im more used to public members being properties rather than fields. I also think its in the c# style guides somewhere.